distribution/receiver/routes/shutdown-route.xml

```diff
@@ -2,8 +2,14 @@
     <route id="shutdown-route">
         <from uri="direct:shutdown-route" />
 
-        <!-- TODO Send a notification to the configured emails -->
-
+      	<doTry>
```

Comment:
I'd think the try catch clause should be in the processor so that you don't duplicate the code in the sender and receiver